doc/api/training/smp_versions/v1.2.0/smd_model_parallel_pytorch.rst

```diff
@@ -0,0 +1,509 @@
+.. admonition:: Contents
+
+   - :ref:`pytorch_saving_loading`
+   - :ref:`pytorch_saving_loading_instructions`
+
+PyTorch API
+===========
+
+**Supported versions: 1.7.1**
+
+This API document assumes you use the following import statements in your training scripts.
+
+.. code:: python
+
+   import smdistributed.modelparallel.torch as smp
+
+
+.. tip::
```

Comment:
Is this not being rendered correctly in markdown

Reply:
yes, its not being correctly rendered in md, but will be fine on the website.